### PR TITLE
Update bare-metal-install* to use swapfile instead of swap partition

### LINKED ISSUE #1131: Revise bare-metal-install-server, -desktop default to swapfile instead of swap partition
**Describe the error/improvement to an existing document or image**
Revise CL OS bare-metal-install-server and bare-metal-install-desktop documentation to reflect revisions to clr-installer: By default use swapfile; make swap partition optional. 

- Align documentation with clearlinux/clr-installer#705
- Swapfile default starts as of release Clear Linux OS 32770.

**Environment (please complete the following):**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by mvincerx**
* Scope applies to `-server` and `-desktop` versions of clr-installer
* Align with clr-installer release PR-705:
* https://github.com/clearlinux/clr-installer/pull/705
* Closes #1131
Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: source/get-started/bare-metal-install-desktop.rst [hunk 1]
```diff
@@ -344,8 +344,10 @@ boot partition
 
 #. Select :guilabel:`Add`.
 
-swap partition
---------------
+swap partition (optional)
+-------------------------
+
+A swapfile is generated by default during installation. However, if you wish to manually create a swap partition, follow the steps below.
```
Comment (mdhorn) [inline source/get-started/bare-metal-install-desktop.rst:350]:
```suggestion
A swapfile is generated by default during installation. However, if you instead wish to create a swap partition, follow the steps below. 
```

File: source/get-started/bare-metal-install-server.rst [hunk 2]
```diff
@@ -318,8 +318,10 @@ boot partition
 
 Now follow the same process to configure the remaining partitions.
 
-swap partition
---------------
+swap partition (optional)
+-------------------------
+
+A swapfile is generated by default during installation. However, if you wish to manually create a swap partition, follow the steps below.
```
Comment (mdhorn) [inline source/get-started/bare-metal-install-server.rst:324]:
```suggestion
A swapfile is generated by default during installation. However, if you instead wish to create a swap partition, follow the steps below. 
```

File: source/get-started/bare-metal-install-desktop.rst [hunk 3]
```diff
@@ -643,16 +645,15 @@ Create partitions per requirements in Table 1.
...
-     - 256MB
-
    * - ``ext[234], XFS, or f2fs``
      - root
      - /
      - *Size depends upon use case/desired bundles.*
 
+.. note:: 
+   
+   A swapfile is generated by default. See `Advanced installation`_
```
Comment (mdhorn) [inline source/get-started/bare-metal-install-desktop.rst:655]:
```suggestion
   A 64MiB swapfile is generated by default. The default size may manually set with the ``--swap-file-size`` command-line option.
```

I don't think it is worth the redirect to the Advanced section there isn't any additional information about the swapfile there. We can add the second sentence about overriding the default swapfile size if you think it is useful to the user.

File: source/get-started/bare-metal-install-server.rst [hunk 4]
```diff
@@ -915,16 +918,15 @@ Create partitions per requirements in Table 1.
...
-     - 256MB
-
    * - ``ext[234], `XFS`, or f2fs``
      - root
      - /
      - *Size depends upon use case/desired bundles.*
 
+.. note:: 
+   
+   A swapfile is generated by default. See `Advanced installation`_
```
Comment (mdhorn) [inline source/get-started/bare-metal-install-server.rst:928]:
```suggestion
   A 64MiB swapfile is generated by default. The default size may manually set with the ``--swap-file-size`` command-line option.
```

I don't think it is worth the redirect to the Advanced section there isn't any additional information about the swapfile there. We can add the second sentence about overriding the default swapfile size if you think it is useful to the user.

File: source/get-started/bare-metal-install-desktop.rst (hunk 1 above)
Comment (karthikprabhuvinod) [reply source/get-started/bare-metal-install-desktop.rst:350]: @mdhorn how about?

A swapfile is generated by default during installation. However, if you instead wish to create a swap partition, check the <Advanced Installation swap link>?

Comment (mvincerx) [reply source/get-started/bare-metal-install-desktop.rst]: So is the word "manually" what's at issue? If so, how about a compromise?:

A swapfile is generated by default during installation. However, if you **prefer** to create a swap partition, follow the steps below. 

"Instead" is already implied by "However, if you prefer..."

File: source/get-started/bare-metal-install-desktop.rst (hunk 3 above)
Comment (mvincerx) [reply source/get-started/bare-metal-install-desktop.rst]: I like your suggestion on adding more details on the `swapfile`.  The link--Advanced installation is not a redirect--but was intended to give users more details on default swapfile and the option to create swap partition if desired. 
I'll defer to your suggestion here.

File: source/get-started/bare-metal-install-server.rst (hunk 2 above)
Comment (mvincerx) [reply source/get-started/bare-metal-install-server.rst]: Same comment as above for `-desktop` version.

File: source/get-started/bare-metal-install-server.rst (hunk 4 above)
Comment (mvincerx) [reply source/get-started/bare-metal-install-server.rst]: Same comment as above for `-desktop` version.

File: source/get-started/bare-metal-install-server.rst [hunk 5]
```diff
@@ -335,6 +338,7 @@ swap partition
 
 #. Press :kbd:`Enter`.
 
+
```
Comment (mdhorn) [inline source/get-started/bare-metal-install-server.rst:341]: Was this extra blank line intentional?

Comment (mvincerx) [reply source/get-started/bare-metal-install-server.rst]: Not intentional. I removed it anyway. It has zero impact on doc builds and CI/CD.

**Comment by mvincerx** [comment]
@karthikprabhuvinod, please reply with approval/changes.

**Pull request merged**